azavea/raster-foundry
# Make the lab work

## Overview

This makes the lab work.

All tools should now be functional.

The front-end still requires some TLC to smooth out the experience.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Notes

It's important that after you make a change to the parameters that you hit the 'Apply Parameters' button. Otherwise it wont work right. This is one of the rough edges.

## Testing Instructions

 * Make sure you have least one project with ingested scenes
 * Go to the lab, try out the tools with the ingested scene project
 * The first render of the tiles may take longer than expected, you may hit cache memory errors
